### ISSUE TITLE
Reapply parts of Bootstrap to printed pages

### ISSUE THREAD

**Issue by jimrandomh** [LessWrong2/Lesswrong2#813]
In the process of ripping out Bootstrap, some CSS that it was applying to printed pages was lost. We want to reapply it (or rather, the subset of it that we actually like).

**Comment by Discordius** [comment]
Is this still an open issue? @jimrandomh?

**Comment by jimrandomh** [comment]
Yes, still open. In particular, Bootstrap has a `!important` rule to prevent text from being gray rather than black, which we haven't replaced.
